### PR TITLE
feat/primary-site-field-optional

### PR DESCRIPTION
**Pull request by kylebcarlson**
* Got a request to make primary_site field optional for users

### CODE REVIEW

**Review by giannif — APPROVED** [review]
👌

File: packages/types/src/gif.ts [hunk 1]
```diff
@@ -71,7 +71,7 @@ export default interface IGif {
     source_tld: string
     source_post_url: string
     is_indexable: boolean
-    is_sticker: boolean
```
Comment (giannif) [inline packages/types/src/gif.ts:74]: @kylebcarlson maybe remove this `is_sticker` change and let's release just the `primary_site` change for now

Comment (kylebcarlson) [reply packages/types/src/gif.ts:74]: sounds good!

**Review by giannif — APPROVED** [review]
<div><br/>via <a href="https://giphy.com/gifs/sean-penn-fast-times-at-ridgemont-high-jeff-spicoli-hZfm9Pj95F9Mk">GIPHY</a></div>

File: packages/types/src/gif.ts (hunk 1 above)
Comment (giannif) [reply packages/types/src/gif.ts:74]: @kylebcarlson can you add in `is_sticker` back? sorry I meant remove the change, not remove the property :)

**Pull request merged**